apps/condo/domains/common/components/Table/Renders.tsx

```diff
@@ -96,7 +132,10 @@ export const getHighlightedContents: TGetHighlightedFN = (search, postfix, extra
     )
 }
 
-const renderLink = (content: JSX.Element | string, href: string) => {
+
+type renderLinkType = (props: { content: JSX.Element | string, href: string }) => React.ReactElement
+
+const renderLink: renderLinkType = ({ content, href }) => {
```

Comment:
it's better to reuse it in our left menu

Reply:
fixed it